<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-8C0LO3K" failures="1" tests="2" name="AdminTest" time="36.393" errors="0" timestamp="2024-03-19T20:28:07 IST">
  <testcase classname="com.obsqura.tests.AdminTest" name="createAdmin" time="8.826">
    <skipped/>
  </testcase> <!-- createAdmin -->
  <testcase classname="com.obsqura.tests.AdminTest" name="createAdmin" time="11.251">
    <failure type="java.lang.AssertionError" message="expected [User Created ] but found [User Created Successfully]">
      <![CDATA[java.lang.AssertionError: expected [User Created ] but found [User Created Successfully]
at com.obsqura.tests.AdminTest.createAdmin(AdminTest.java:25)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- createAdmin -->
</testsuite> <!-- AdminTest -->
